.landscape.yml

```diff
@@ -0,0 +1,2 @@
+python-targets:
```

Comment:
I don't think this is needed. [This should be enough](https://github.com/ActivityWatch/aw-core/blob/f515e83b76d14dd7fa01722b25bb6d06358197f4/setup.py#L24).